executor/adapter.go

```diff
@@ -217,7 +217,6 @@ func (a *ExecStmt) PointGet(ctx context.Context, is infoschema.InfoSchema) (*rec
 		}
 		a.PsStmt.Executor = newExecutor
 	}
-	stmtNodeCounterSelect.Inc()
```

Comment:
Seems `StmtCtx.StmtType` should be set in this path if this line is removed

Reply:
So I add https://github.com/pingcap/tidb/pull/17560/files#diff-928cb614ae092ed4c6f349f4c4c991e4R1412 for all prepared statements.

Reply:
Maybe it's difficult for users to understand `CPS`, or what a `command` means?

Reply:
emm... yes, but I'm not sure if we should provide more details in the description of the metric.

Reply:
Add more descriptions...

Reply:
Do we need another panel to show the real `QPS` or just use `Statement OPS`

Reply:
I used to try to add a new panel, but there're too many panels in TiDB dashboard.